addons/binding/org.openhab.binding.hyperion/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hyperion" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="server">
+		<label>Hyperion Server</label>
+		<description>This is a Hyperion server</description>
+		<channels>
+			<channel id="brightness" typeId="brightness" />
+			<channel id="color" typeId="color" />
+			<channel id="effect" typeId="effect" />
+			<channel id="clear" typeId="clear" />
+			<channel id="clear_all" typeId="clear_all" />
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host address</label>
+				<description>The host address of the Hyperion server.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
+				<label>Port</label>
+				<description>The port of the Hyperion server.</description>
+				<required>true</required>
+				<default>19444</default>
+			</parameter>
+			<parameter name="priority" type="integer" required="true">
+				<label>Priority</label>
+				<description>The priority associated with updates sent to the Hyperion server.</description>
+				<required>true</required>
+				<default>50</default>
+			</parameter>
+			<parameter name="poll_frequency" type="integer" required="true">
+				<label>Polling frequency</label>
+				<description>How often (in seconds) to poll the Hyperion server for value changes.</description>
+				<required>true</required>
+				<default>15</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<!-- Sample Channel Type -->
+	<channel-type id="brightness">
+		<item-type>Dimmer</item-type>
+		<label>Brightness</label>
+		<description>Sets the brightness of the LEDs</description>
+	</channel-type>
+	<channel-type id="color">
+		<item-type>Color</item-type>
+		<label>Color</label>
+		<description>Sets the color of the LEDs</description>
+	</channel-type>
+
+	<channel-type id="effect">
+		<item-type>String</item-type>
+		<label>Effect</label>
+		<description>Sets the effect of the LEDs</description>
```

Comment:
you should definitely also provide a state description with the possible values here.